forte/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -359,7 +382,7 @@ double SADSRG::compute_reference_energy_from_ints() {
     V.block("aaaa")("prqs") =
         ints_->aptei_ab_block(actv_mos_, actv_mos_, actv_mos_, actv_mos_)("prqs");
 
-    Eref_ = compute_reference_energy(H, Fock_, V);
+    Eref_ = compute_reference_energy(H, Fock_, V) + ints_->scalar();
```

Comment:
Please add a comment about why the scalar may be nonzero. If this is embedding-specific, it won't be clear to most people reading the code. If this is not embedding-specific, you've changed the behavior of `Eref_`. I can't tell which case we're in.

Reply:
In general situations this scalar should be 0. In embedding cases, however, since we build the fragment integrals from empty, the final computations do not have information about frozen core energy and nuclear repusion energy, we have to added them to the final results.